cluster/images/provider-kafka-controller/Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM BASEIMAGE
+FROM gcr.io/distroless/static@sha256:d2b0ec3141031720cf5eedef3493b8e129bc91935a43b50562fbe5429878d96b
 
-COPY package.yaml .
+ARG ARCH
+ARG TINI_VERSION
```

Comment:
This can be removed since we don't use tini here 👍🏻 
```suggestion
```

Reply:
Removed. Also updated to use `TARGETOS/TARGETARCH` now.